lib/presentation/widgets/m3_widgets/cards/outlined_trip_card.dart

```diff
@@ -108,27 +105,11 @@ class OutlinedTripCard extends StatelessWidget {
               ],
             ),
           ),
-          if (areThereBottomAction ?? true) const Divider(),
-          if (areThereBottomAction ?? false)
-            Padding(
-              padding: const EdgeInsets.only(
-                left: AppDimensions.d16,
-                right: AppDimensions.d16,
-                bottom: AppDimensions.d10,
-              ),
-              child: Row(
-                mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                children: [
-                  WarningOutlinedButton(
-                    text: 'End the trip',
-                    onPressed: () {},
-                  ),
-                  PrimaryElevatedButton(
-                    text: 'Add expense',
-                    onPressed: () => context.router.push(CreateExpensesRoute(tripId: '', currency: '')),
-                  ),
-                ],
-              ),
+          if (withActionButtons) const Divider(),
+          if (withActionButtons == true)
+            OutlinedTripCardButtons(
+              onPressedWarningOutlinedButton: () {},
+              onPressedPrimaryElevatedButton: () {},
             ),
```

Comment:
merge this with list and one condition because it's the same. When you try to say that something is true you dont need to write `== true`, calling the variable already means `true`